test/test_custom_ops.py

```diff
@@ -4147,6 +4148,131 @@ def test_any_output_is_alias_to_input_or_output(self):
             )
         )
 
+    def test_library_get_kernel(self):
+        """Test registering a custom kernel, using it, then deregistering and verifying error."""
+
+        # Register a dummy kernel for arange to the CPU key that returns a tensor of ones
+        def dummy_arange_cpu(
+            start,
+            end,
+            dtype=None,
+            layout=torch.strided,
+            device=None,
+            pin_memory=False,
+        ):
+            size = max(0, int(end - start))
+            return torch.ones(size, dtype=dtype, device=device)
+
+        with torch.library._scoped_library("aten", "IMPL") as lib:
+            lib.impl("arange.start", dummy_arange_cpu, "CPU")
+
+            kernel = torch.library.get_kernel("aten::arange.start", "CPU")
+            op_handle = torch.ops.aten.arange.start._handle
+            dispatch_keys = torch._C.DispatchKeySet(torch._C.DispatchKey.CPU)
```

Comment:
Something I'm not too sure about is how we expect users of this API to construct the DispatchKeySet to pass to `kernel.call_boxed`